### PR TITLE
Fix element_multiply with magma

### PR DESCRIPTION
**Pull request by jeanlucf22**

### CODE REVIEW

**Review by suemni — APPROVED** [review]

**Review by nicolasbock — APPROVED** [review]
LGTM

**Comment by mewall** [comment]
Thanks for fixing this! 

I have a couple of PRs to make as well related to the magma path, will wait until this is merged so I can rebase on top of this first.

**Review by cnegre — APPROVED** [review]

**Pull request merged**
